.github/workflows/ci-for-default-ns.yaml

```diff
@@ -75,15 +75,18 @@ jobs:
           # for testing start-debug and stop-debug deployment
           # mons
           kubectl rook_ceph debug start rook-ceph-mon-a
-          sleep 5
+          tests/github-action-helper.sh wait_for_condition "rook-ceph-mon-a-debug deployment pod to start" 10 \
```

Comment:
Okay
Removed the starting wit time

Reply:
No, but why do we need three lines for the caller? Lines 78-80 should **only** be one line:
```
tests/github-action-helper.sh wait_for_running_pod rook-ceph-mon-a-debug
```
 If the caller is a single line instead of three, it is much simpler. I don't mean to create a very long line out of three, I mean that the one line is very simple and everything else for the implementation can be in the `wait_for_running_pod` function.

Reply:
`osd=$(kubectl get pod -n test-cluster | grep rook-ceph-osd-0 | awk '!/rook-ceph-osd-0-debug/' | awk '{ print $1  }')` 

One command was this, so do not pick the debug pod that is stopping,

Should I use if else to manage this in the function?

Reply:
Perhaps we need a second argument to the function such as this? Then the function could handle it with if/else
```
tests/github-action-helper.sh wait_for_pod rook-ceph-mon-a-debug [running | deleted]
```

Reply:
The 3rd argument, 2nd I already used for namespace XD